.github/workflows/run-precommit.yml

```
@@ -0,0 +1,31 @@
+name: Run pre-commit hooks
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  pre-commit:
+    strategy:
+      matrix:
+        py_version: ["3.8", "3.9", "3.10"]
```

Let's just remove 3.8 now:
```suggestion
        py_version: ["3.9", "3.10"]
```

If you agree, could you also update the README and TOML to set the required minimum version to 3.9?